Autopilot/Simulation/Intercepts/CommsWithTelemetry_intercept_user.cpp

```diff
@@ -0,0 +1,31 @@
+#include "CommsWithTelemetry.hpp"
+
+/********************************************************************
+* Variables
+********************************************************************/
+
+static Telemetry_PIGO_t userPigo[100];
+
+/********************************************************************
+* Code
+********************************************************************/
+
+void CommWithTelemInit()
+{
+}
+
+void SendPathData(POGI *data)
+{
+ (void) data;
```

Comment:
How does this compile?

Reply:
why wouldn't it ? (void) <insert arg here> is just a way to tell the compiler that we're not using that arg. It avoids us getting a warning.

Reply:
Ah fair enough lol. I should've phrased that question nicer. Sorry